openmc/data/neutron.py

```diff
@@ -931,3 +931,27 @@ def _get_redundant_reaction(self, mt, mts):
         rx.redundant = True
 
         return rx
+
+    def check_for_reaction(self, mt):
+        """Determine minimal set of MT numbers for this reaction and library.
+
+        Parameters
+        ----------
+        mt : int
+            ENDF MT number of the reaction to look for or find components of.
+
+        Returns
+        -------
+        mts : list of int
+            ENDF MT numbers of reactions that make up the redundant reaction and
+            have cross sections provided.
+
+        """
+        mts = []
+        if mt in SUM_RULES and mt not in self:
+            for mt_i in SUM_RULES[mt]:
+                mts += self.get_reaction_components(mt_i)
+        if mts:
+            return mts
+        else:
+            return [mt] if mt in self else []
```

Comment:
Two comments on this:
- Since the only difference between this and `get_reaction_components` is whether the recursion occurs when an MT number is already found, it seems like we ought to just add an optional boolean argument on `get_reaction_components` that allows for that.
- I'm not really understanding why the change is needed in the first place. Won't `get_reaction_components` already work just fine if used in plotter.py?

Reply:
With regards to point 1 - yes that's definitely a better way of doing it. With regards to point 2,  I suppose it doesn't need to be implemented if when you ask to plot the total cross section you want it to always return the sum of all the MT values that comprise it rather than just  2 + 3 if those explicitly exist in the library. Ideally these two options return the same value. I'm fine with doing it this way if that's the behavior you would prefer.

Reply:
Yes, generally the preference is to always use the lowest-level reactions when possible in the event that there are conflicts. Of course, this shouldn't happen, but my guess is that if the cross section for a redundant reaction doesn't agree with the sum of its constituents, it's probably the redundant one that's wrong.

Reply:
Ok sounds good. I'll revert these changes and just use `get_reaction_components`